Description: Load system-wide application defaults from /etc/X11/app-defaults
 Section 11.8.6 of the Debian policy requires all application defaults to be
 stored under /etc/X11/app-defaults. 
Forwarded: not-needed
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Bug-Debian: http://bugs.debian.org/348697
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-02-24
Index: rxvt-unicode/src/rxvttoolkit.C
===================================================================
--- rxvt-unicode.orig/src/rxvttoolkit.C	2015-01-20 13:49:52.650045908 -0500
+++ rxvt-unicode/src/rxvttoolkit.C	2015-01-20 13:49:52.650045908 -0500
@@ -348,13 +348,15 @@
   // 6. System wide per application default file.
 
   /* Add in $XAPPLRESDIR/Rxvt only; not bothering with XUSERFILESEARCHPATH */
-  if ((xe = getenv ("XAPPLRESDIR")))
+  if (!(xe = getenv ("XAPPLRESDIR")))
     {
-      snprintf (fname, sizeof (fname), "%s/%s", xe, RESCLASS);
-
-      if ((rdb1 = XrmGetFileDatabase (fname)))
-        XrmMergeDatabases (rdb1, &database);
+      // Debian-specific; see #348697 and Policy
+      xe = "/etc/X11/app-defaults";
     }
+  snprintf (fname, sizeof (fname), "%s/%s", xe, RESCLASS);
+
+  if ((rdb1 = XrmGetFileDatabase (fname)))
+    XrmMergeDatabases (rdb1, &database);
 
   // 5. User's per application default file.
   // none
